src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(param) {
+      calculator.result += param;
+    },
+
+    subtract(param) {
+      calculator.result -= param;
+    },
+
+    multiply(param) {
+      calculator.result *= param;
+    },
+
+    divide(param) {
+      calculator.result /= param;
```

Comment:
The `divide` method does not handle division by zero. This can lead to runtime errors or unexpected results. Consider adding a check to ensure `param` is not zero before performing the division.